addons/l10n_latam_check/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_account_payment_mass_transfer,access_account_payment_mass_transfer,model_account_payment_mass_transfer,account.group_account_invoice,1,1,1,0
+l10n_latam_checkbook_access_full,l10n_latam_checkbook_access_full,model_l10n_latam_checkbook,account.group_account_invoice,1,1,1,1
+l10n_latam_checkbook_access_global,l10n_latam_checkbook_access_global,model_l10n_latam_checkbook,base.group_user,1,1,1,1
```

Comment:
Giving full access to everybody and also to billing users?
Anyone can change the sequence?

Reply:
fixed, only read access